kritis-charts/templates/kritis-server-webhook.yaml

```diff
@@ -16,7 +16,6 @@ webhooks:
           - pods
     failurePolicy: Fail
     clientConfig:
-      caBundle: # TODO: Add script to inject caBundle here
       service:
```

Comment:
I think we need to add `caBundle: {{ .Values.caBundle}}` here?

Reply:
yea sorry, missed this switching things over. added now